### DISCUSSION TITLE
How can I wrap words in g6-react-node <Text> like CSS word-wrap

### DISCUSSION THREAD

**Discussion by tbuddhi-dev** [antvis/G6#4798] [Q&A] [answered]
When create a custom node using g6-react-node in antv/g6, how can I break text lines to multiple lines when it's exceed the max width of node like CSS word-wrap?

I hope someone can help me on this. Thank you.
`<Group>
 <Rect
  style={{
   width: 100,
   height: 70, 
   fill: 'l(0) 0:#D67200 1:#FABD55',
   radius: 6,
  }}
 >
  <Text 
   style={{
    fill: "#FFF", 
    maxWidth: 100, 
    display: 'flex',
    flexWrap: 'wrap'
   }} 
  >
  {label}
  </Text>
 </Rect>
</Group>`

**Comment by hustcc** [comment, marked as answer]
新版本 G6 5.0 才支持，因为 maxWidth 需要底层 G 5.0 才有的。

另外，在 G6 5.0 版本中，还可以使用 React 去自定义节点。
